apps/web/config/handler.ts

```diff
@@ -1,10 +1,73 @@
 import Axios from 'axios';
+import { ENV } from './ENV';
 
 export const apiHandler = Axios.create({
-    baseURL: process.env.NEXT_PUBLIC_API_URL,
+    baseURL: ENV.api_base + '/api',
     withCredentials: true,
     headers: {
         Accept: 'application/json',
         'Content-Type': 'application/json',
     },
 });
+
+apiHandler.interceptors.response.use(
+    (response) => {
+        return response;
+    },
+    async function (error) {
+        const originalRequest = error.config;
+        if (
+            error.response.status === 401 &&
+            originalRequest.url === ENV.api_base + '/auth/refresh'
+            // redirecting when the refresh call fails
+        ) {
+            window.location.href = '/';
+            return Promise.reject(error);
+        }
+
+        if (
+            error.response.data.code === 'token_not_valid' &&
+            error.response.status === 401 &&
+            error.response.statusText === 'Unauthorized'
+        ) {
+            const refreshToken = localStorage.getItem('refresh_token');
```

Comment:
Why are you storing & retrieving refresh token from localStorage?
It's in the cookie.